legacy/rise_6dof/src/rise_6dof/controller.py

```diff
@@ -60,10 +66,12 @@ def update(self, dt, desired, current):
 
         error_velocity_world = (desired_x_dot + body_gain(
             numpy.diag(self.config['k'])).dot(error_position_world)) - x_dot
+        print('{:20s} {:30}'.format('Error velocity:', error_velocity_world))
```

Comment:
Ks is velocity error, this is starting to make sense. I like the print statements btw, very nice.